src/SqlPersistence.Tests/Saga/OracleSagaPersisterTests.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+using System.Data.Common;
+using NServiceBus.Persistence.Sql.ScriptBuilder;
+using NUnit.Framework;
+
+[TestFixture]
+public class OracleSagaPersisterTests : SagaPersisterTests
+{
+    public OracleSagaPersisterTests() : base(BuildSqlVariant.Oracle, null)
+    {
+    }
+
+    protected override Func<DbConnection> GetConnection()
+    {
+        return () =>
+        {
+            var connection = OracleConnectionBuilder.Build();
+            connection.Open();
```

Comment:
It matches the MsSQL/MySQL saga test classes. Just based upon how the SagaPersisterTests are set up in comparison to the other persisters.